src/array/ops/image.rs

```diff
@@ -179,6 +226,77 @@ impl ImageArray {
         array.as_ref().as_any().downcast_ref().unwrap()
     }
 
+    pub fn from_vecs<T: arrow2::types::NativeType>(
```

Comment:
@samster25 I consolidated the kernels and casting vecs --> ImageArray construction to this single function, to make sure that those paths don't diverge.